redash/settings/__init__.py

```diff
@@ -107,6 +107,8 @@ def all_settings():
 COOKIE_SECRET = os.environ.get("REDASH_COOKIE_SECRET", "c292a0a3aa32397cdb050e233733900f")
 SESSION_COOKIE_SECURE = parse_boolean(os.environ.get("REDASH_SESSION_COOKIE_SECURE") or str(ENFORCE_HTTPS))
 
+SECRET_KEY = os.environ.get('REDASH_SECRET_KEY', COOKIE_SECRET)
```

Comment:
I would take the opportunity to make sure that this is a non empty value since this is a [critical setting in Flask](http://flask.pocoo.org/docs/1.0/config/#SECRET_KEY).

Reply:
I realized that this is 1) beyond the scope of the PR; 2) I don't have the time to implement it :-(. To avoid having this PR lingering any longer, went ahead without it.

Reply:
Makes sense.